### PR TITLE
Home Owners styling and Transfer Bugs

### PR DESCRIPTION
**Pull request by cameron-eyds**
*Issue #:* /bcgov/entity#17754  /bcgov/entity#18197

*Description of changes:*
* Fix to remove groups that had only added owners after a 'previous' owner mark for removal
* Fix for owner group if it was edited before removing current owners, will now convert to an added group
* Styling changes to Home Owners to restore spacing and appearances

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the PPR license (Apache 2.0).

### CODE REVIEW

File: ppr-ui/package.json [hunk 1]
```diff
@@ -1,13 +1,13 @@
...
-  "version": "3.0.6",
+  "version": "3.0.7",
   "private": true,
   "appName": "Assets UI",
   "sbcName": "SBC Common Components",
   "scripts": {
     "dev": "vite dev",
     "build": "vite build",
-    "serve": "vite preview --port 8080",
+    "preview": "vite preview --port 8080",
```
Comment (dimak1) [inline ppr-ui/package.json:10]: Where would this be used?

File: ppr-ui/src/assets/styles/base.scss [hunk 2]
```diff
@@ -229,6 +233,11 @@ a {
   width: 0;
 }
 
+.actions {
+  padding-right: 28px!important;
```
Comment (dimak1) [inline ppr-ui/src/assets/styles/base.scss]: Is there a way to avoid using important, as this is not the best practice.

**Review by dimak1 — APPROVED** [review]

File: ppr-ui/package.json (hunk 1 above)
Comment (cameron-eyds) [reply ppr-ui/package.json:10]: When you want to preview the production ready build

**Review by RuoxuanPengBC — APPROVED** [review]
Looks great

File: ppr-ui/src/assets/styles/base.scss (hunk 2 above)
Comment (cameron-eyds) [reply ppr-ui/src/assets/styles/base.scss]: Is there a way? Always.

I will explore some options.

**Review by doug-lovett — APPROVED** [review]
Excellent.

File: ppr-ui/src/assets/styles/base.scss (hunk 2 above)
Comment (cameron-eyds) [reply ppr-ui/src/assets/styles/base.scss]: Updated 👍

Comment (dimak1) [reply ppr-ui/src/assets/styles/base.scss]: Great, thank you!

**Pull request merged**